aten/src/ATen/Context.h

```diff
@@ -432,9 +432,9 @@ class TORCH_API Context {
   std::array<at::SDPBackend, at::num_sdp_backends> sdp_priority_order = {
       at::SDPBackend::flash_attention,
       at::SDPBackend::efficient_attention,
+      at::SDPBackend::overrideable,
```

Comment:
can you undo this change, this code
https://github.com/pytorch/pytorch/blob/731ee31f7b6ba19307daab323f6196172b71aaf8/aten/src/ATen/native/transformers/cuda/sdp_utils.cpp#L60-L100

shows how to set a order per a specific backend

Reply:
Thanks, it looks good. We will follow this code to set XPU priority order. cc @LuFinch

Reply:
@drisspg May I know if we have addressed your comments.